internal/util/kms.go

```diff
@@ -0,0 +1,242 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// kmsProviderKey is the name of the KMS provider that is registered at
+	// the kmsManager. This is used in the ConfigMap configuration options.
+	kmsProviderKey = "KMS_PROVIDER"
+	// kmsTypeKey is the name of the KMS provider that is registered at
+	// the kmsManager. This is used in the configfile configuration
+	// options.
+	kmsTypeKey = "encryptionKMSType"
+
+	// podNamespace ENV should be set in the cephcsi container
```

Comment:
@nixpanic why do we need this variable of podNamespace ?

Reply:
It is used by the Hashicorp Vault Tokens KMS when it is configured with a ConfigMap. This PR just moves it to this file.